### DISCUSSION TITLE
Child dependency missing in node_modules

### DISCUSSION THREAD

**Discussion by arjun11120** [pnpm/pnpm#6331] [General]
Migration from npm to pnpm child dependency missing in node_modules.Using PNPM the react-dropzone in package.json only create the react-dropzone in node_modules. It doesn't create the dropzone in node_modules. How can I fix it ?

**Comment by lvqq** [comment]
See https://pnpm.io/symlinked-node-modules-structure. 

The child dependencies are in the `node_modules/.pnpm` directory by default. If you want to hoist them, you can try [hoist](https://pnpm.io/npmrc#hoist) configuration
